DolbyIO/Source/Public/DolbyIOFunctions.h

```diff
@@ -368,12 +368,12 @@ class UDolbyIOBlueprintFunctionLibrary : public UBlueprintFunctionLibrary
 	 * This function should be called before the first call to Set Token if the user needs logs about the plugin's
 	 * operation. Calling more than once has no effect.
 	 *
-	 * @param SdkLogLevel - Log level for SDK logs. Default is Info.
-	 * @param MediaLogLevel - Log level for Media Engine logs. It is recommended to keep the Media Engine log level
-	 * at OFF, ERROR or WARNING to avoid spam and only enable more detailed logs when necessary. Default is Off.
-	 * @param LogDirectory - Directory to which the logs should be saved. The application must have write access to
+	 * @param SdkLogLevel - Log level for SDK logs. The default value is Info.
+	 * @param MediaLogLevel - Log level for Media Engine logs. We recommend keeping the Media Engine log level
+	 * at OFF, ERROR, or WARNING to avoid spam and only enable more detailed logs when necessary. The default value is OFF.
```

Comment:
To be 100% correct, you may consider changing OFF to Of, ERROR to Error and WARNING to Warning.